code/controllers/subsystem/processing/quirks.dm

```diff
@@ -11,12 +11,14 @@ PROCESSING_SUBSYSTEM_DEF(quirks)
 	runlevels = RUNLEVEL_GAME
 	wait = 1 SECONDS
 
-	var/list/quirks = list() //Assoc. list of all roundstart quirk datum types; "name" = /path/
-	var/list/quirk_points = list() //Assoc. list of quirk names and their "point cost"; positive numbers are good traits, and negative ones are bad
-	///An assoc list of quirks that can be obtained as a hardcore character, and their hardcore value.
+	/// Assoc. list of all roundstart quirk datum types; "name" = /path/
+	var/list/quirks = list()
+	/// Assoc. list of quirk names and their "point cost"; positive numbers are good traits, and negative ones are bad
+	var/list/quirk_points = list()
+	/// An assoc list of quirks that can be obtained as a hardcore character, and their hardcore value.
 	var/list/hardcore_quirks = list()
 
-	/// A list of quirks that can not be used with each other. Format: list(quirk1,quirk2),list(quirk3,quirk4)
+	/// A list of quirks that can not be used with each other. Format: list(quirk1,quirk2), list(quirk3,quirk4)
```

Comment:
```suggestion
	/// A list of quirks that can not be used with each other
```

Having a format in the code comments is really dumb, especially since it's just a basic list.